src/site/stages/build/process-cms-exports/transformers/paragraph-staff_profile.js

```diff
@@ -1,9 +1,41 @@
+const { getImageCrop } = require('./helpers');
+
+const setImageCrop = media => {
+  const imageObj = Object.assign({}, media);
+
+  // Reset the values to the original state
+  // image: {
+  //   url: encodeURI(entity.thumbnail[0].url.replace('public:/', '/img')),
+  //   derivative: {
+  //     url: encodeURI(entity.thumbnail[0].url.replace('public:/', '/img')),
+  //     width: entity.image[0].width,
+  //     height: entity.image[0].height,
+  //   },
+  // },
+  imageObj.image.url = encodeURI(
+    imageObj.thumbnail.url.replace('public:/', '/img'),
+  );
+  imageObj.image.derivative.url = encodeURI(
+    imageObj.thumbnail.url.replace('public:/', '/img'),
+  );
+  imageObj.image.derivative.width = 0;
+  imageObj.image.derivative.height = 0;
+
+  // Re-apply the Image style size
+  return getImageCrop(imageObj, '_1_1_SQUARE_MEDIUM_THUMBNAIL');
```

Comment:
Can we move this logic into `getImageCrop`?

Reply:
No, because this only applies to this transformer. This is already happening by default to all other nodes.

Reply:
>  this only applies to this transformer

That we know of. 😄 

But OK - if we find more cases we can always refactor later.